flutter_application_1/lib/main.dart

```diff
@@ -1,7 +1,34 @@
+//Ali's Change
 import 'package:flutter/material.dart';
+//import 'package:permission_handler/permission_handler.dart';
+//import 'package:location/location.dart';
```

Comment:
Does the [AndroidManifest.yml](https://github.com/AElga/GhostPeerShareApp/blob/main/flutter_application_1/android/app/src/main/AndroidManifest.xml) affect these permission grants?

Looks like MacOS is not [supported](https://pub.dev/packages/permission_handler)